server/data/notificationsApiClient.ts

```diff
@@ -51,19 +51,22 @@ class NotificationsApiClient {
     visitTime,
     visitDate,
     prisonPhoneNumber,
+    reference,
   }: {
     phoneNumber: string
     prisonName: string
     visitTime: string
     visitDate: string
     prisonPhoneNumber: string
+    reference: string
   }): Promise<SmsResponse> {
     return this.notificationsApiClient.sendSms(templates.cancellationConfirmation, phoneNumber, {
       personalisation: {
         prison: prisonName,
         time: visitTime,
         date: visitDate,
         'prison phone number': prisonPhoneNumber,
+        reference,
```

Comment:
This means it will need to be referenced in the Notify as `((reference))` rather than `((ref number))` like the others. Doesn't particularly matter - just need to be aware and use the right one. Personally I think it's better not to use spaces in these 🤷‍♂️